src/datanode/src/instance.rs

```diff
@@ -315,11 +315,21 @@ pub(crate) async fn new_object_store(store_config: &ObjectStoreConfig) -> Result
         ObjectStoreConfig::Oss { .. } => new_oss_object_store(store_config).await,
     };
 
+    // Don't enable retry layer when using local file backend.
+    let object_store = if !matches!(store_config, ObjectStoreConfig::File(..)) {
+        object_store.map(|object_store| object_store.layer(RetryLayer::new().with_jitter()))
+    } else {
+        object_store
+    };
+
     object_store.map(|object_store| {
         object_store
-            .layer(RetryLayer::new().with_jitter())
             .layer(MetricsLayer)
-            .layer(LoggingLayer::default())
+            .layer(
+                LoggingLayer::default()
+                    // Print expected error only in DEBUG level.
+                    .with_error_level(Some(log::Level::Debug)),
```

Comment:
Do we expect to inhibit all error logs from opendal?

Reply:
And https://docs.rs/opendal/latest/opendal/layers/struct.LoggingLayer.html#method.with_error_level

Only setting the log level while the expected error happened.